app/account/action/update-image-metadata.ts

```diff
@@ -0,0 +1,46 @@
+'use server'
```

Comment:
Is this file exactly the same code as the file under the trip action directory?  If so, it is duplicated so could you merge it? It is better for us to use one action file throughout any features/

Reply:
it's actually slightly different!